doc/api/net.markdown

```diff
@@ -60,40 +60,17 @@ Use `nc` to connect to a UNIX domain socket server:
 ## net.connect(options[, connectionListener])
 ## net.createConnection(options[, connectionListener])
 
-A factory method, which returns a new ['net.Socket'](#net_class_net_socket)
-and connects to the supplied address and port.
+A factory function, which returns a new ['net.Socket'](#net_class_net_socket)
+and automatically connects with the supplied.
```

Comment:
I'm not a native english speaker, but after you remove `address and port` this doesn't make much sense to me anymore.